bash_completions.go

```diff
@@ -29,14 +30,6 @@ __debug()
     fi
 }
 
-# Homebrew on Macs have version 1.3 of bash-completion which doesn't include
-# _init_completion. This is a very minimal version of that function.
-__my_init_completion()
-{
-    COMPREPLY=()
-    _get_comp_words_by_ref cur prev words cword
-}
```

Comment:
Fixed. Actually, we call _init_completion already with `-s` which implies `-n =`. Only on the Mac, I had to add `-n =` manually.